envtpl.py

```diff
@@ -120,15 +125,23 @@ def stdout_write(output):
         io.TextIOWrapper(sys.stdout.buffer, encoding='utf-8').write(output)
 
 
-def _render_file(filename, variables, undefined):
+def _render_file(filename, variables, undefined, var_delimiter):
     dirname = os.path.dirname(filename)
     loader = jinja2.FileSystemLoader(dirname)
     relpath = os.path.relpath(filename, dirname)
-    return _render(relpath, loader, variables, undefined)
+    return _render(relpath, loader, variables, undefined, var_delimiter)
 
 
-def _render(template_name, loader, variables, undefined):
-    env = jinja2.Environment(loader=loader, undefined=undefined)
+def _render(template_name, loader, variables, undefined, var_delimiter):
```

Comment:
Would it be possible to add validation for `var_delimiter`, e.g. ensuring it has at most one comma, that neither start nor strings are empty, etc.?